docs/PowerDistribution.md

```diff
@@ -0,0 +1,99 @@
+# AMDC Power Distribution
+
+This document describes the design considerations and implementation details for the power distribution subsystem on the Advanced Motor Drive Controller (AMDC). A block diagram is presented and discussed in detail, along with PCB layout considerations.
+
+## Design Requirements
+
+The design requirements for the power input and distribution on the AMDC can be boiled down to a succient list of conflicting goals. The remainder of this document will summarize the resulting AMDC power system design and how it adheres to these goals. The following describes these goals:
+
+1. Single voltage input connection for all PCB power.
+2. Efficient DC/DC conversion for various voltage rails (steady state should not be excessively hot for devices).
+3. Low noise on power rails for circuitry (switching DC/DC are noisy; LDOs are preferred).
+4. Power subsystem is not the main goal of AMDC, therefore BOM should be small and inexpensive.
+
+## Block Diagram
+
+<img src="images/amdc-power-distribution.svg" />
```

Comment:
Questions about this figure:

- What are the `OR` blocks? Are these each LDO's? If so, perhaps the pink blocks should reflect the trimmed output voltage of the DC/DC regulator? 

- I am confused about what the pink blocks represent, they seem to have the same name as the text on the right side of the image. 

- There are several gray blocks that look like switches. What are these? Jumpers?

Reply:
I have updated the block diagram, hopefully it is more clear now.